packages/usdk/packages/upstreet-agent/packages/react-agents/classes/discord-manager.ts

```diff
@@ -411,11 +466,71 @@ export class DiscordBot extends EventTarget {
       });
     };
 
+    // message reactions
+    const _bindIncomingMessageReactions = () => {
+      const handleReaction = (e: MessageEvent, eventType: string) => {
+        const {
+          userId,
+          messageId,
+          emoji,
+          channelId,
+          userDisplayName,
+        } = e.data;
+
+        console.log(eventType, {
+          userId,
+          userDisplayName,
+          messageId,
+          emoji,
+          channelId,
+        });
+
+        // look up conversation
+        const conversation = this.dmConversations.has(userId)
+          ? this.dmConversations.get(userId) ?? null
+          : this.channelConversations.get(channelId) ?? null;
+
+        if (!conversation) return;
+
+        const rawMessageReaction = {
+          userId,
+          name: userDisplayName,
+          method: 'messageReaction',
+          args: {
+            reaction: emoji,
+            messageId,
+            userId,
+            context: {
```

Comment:
Is this used?

Reply:
updated to remove the args, and constructing the context with emoji and messageId for the reaction